cln-grpc/build.rs

```diff
@@ -1,6 +1,7 @@
 fn main() {
     let builder = tonic_build::configure();
     builder
+        .type_attribute(".", "#[derive(serde::Serialize,serde::Deserialize)]")
```

Comment:
I take it that the desired serialization format is JSON? In that case, how are binary values being handled?

Reply:
Yes, we'd like to be able to de-serialize to JSON. It makes pretty-printing the payload for advanced users easier (e.g. see dev command result).

The binary fields are then printed as hex vectors.